Fix  E201, E202, and E225 flake8 warnings

sdudoladov opened the pull request:
Hi all,

This PR fixes `flake8` warnings

- `E201: whitespace after '('`
- `E202: whitespace before ')'`
- `E225: missing whitespace around operator`

mentioned in #5599 

A quick test of running `flake8 --select E201,E202,E225` in the project's  root directory produces no more errors.

Totktonada commented:
Thank you for the patchset!

Several general changes are required before we'll dive into the review:

1. A patch set must not contain temporary commits to squash, merge commits and so on. Otherwise responsibility to write an explanatory commit message would be on a maintainer: it would be too much work for maintainers.
2. All commits of the patchset should be have proper description and follow our [commit message style](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message).
3. If there are conflicts with recent `master`, the patchset should be rebased on top of `master`. Don't hesitate: you **can** and ever must force-push your developer branch.
4. A change should be accompanied with a regression test. In this case it is flake8 config as least, enabling into CI at most.

Regarding the config: it surely will blacklist certain warnings. So it should contain a link to the issue, where those warnings supposed to be fixed.

sdudoladov commented:
@Totktonada Thanks for the detailed feedback. I've added the `.flake8` configuration file, rebased and squashed  the commits and re-written the commit message.

Totktonada commented:
1. The submodules should not be updated here.
2. `.flake8` whitelists warnings, not blacklists them. So we'll miss everything except those three fixed points.
3. It worth to add a CI rule to enforce the check. Otherwise further changes may return us back.

   I would rename `.github/workflows/luacheck.yml` to `.github/workflows/linters.yml` and add a job with flake8 here. @avtikhon, what do you think?

kyukhin commented:
More than 6 months w/o reply. Closing.
